google/cloud/storage/client.h

```diff
@@ -1425,6 +1450,13 @@ class Client {
   /**
    * Updates the value of an existing bucket ACL.
    *
+   * @note
+   * For changing BucketAccessControl the Patch and Update APIs basically offer
```

Comment:
Opt: I might put a comma after "For changing BucketAccessControl" (et seq).

Reply:
Done.